contracts/EmailRecoveryCommandHandler.sol

```diff
@@ -131,10 +131,6 @@ contract EmailRecoveryCommandHandler is IEmailRecoveryCommandHandler {
             revert InvalidAccount();
         }
 
-        if (recoveryModuleInEmail != address(this)) {
-            revert InvalidRecoveryModule();
-        }
-
```

Comment:
I removed these lines because commandHandler address is not recoveryModuleAddress in email template.